### PR TITLE
[W5.4][T11-A4] Philemon Tan 

### PR DESCRIPTION
**Pull request by philemontan**

### CODE REVIEW

**Comment by tanchc** [comment]
Could try having more than one sort order so that users have more freedom in choosing how they want their data sorted.

**Comment by eugenepeh** [comment]
@philemontan gentle reminder, remember to update the user guide and include some junit tests.

**Comment by philemontan** [comment]
User guide added previously in commit 830ed35 already.

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts all people by alphabetical order" +
+            "No parameter needed.";
+
+    public static final String MESSAGE_SUCCESS = "All people sorted";
+    public static final String MESSAGE_ALREADY_SORTED = "Already sorted";
+
+
+    @Override
+    public CommandResult execute(){
```
Comment (eugenepeh) [inline src/seedu/addressbook/commands/SortCommand.java]: you've forgotten the space before left curly

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (eugenepeh) [inline src/seedu/addressbook/parser/Parser.java]: remember to disable intellij auto import wildcard
[Why is wild card import bad?](https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad)

File: test/java/seedu/addressbook/commands/SortCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+import java.util.Iterator;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class SortCommandTest {
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts all people by alphabetical order" +
+            "No parameter needed.";
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/commands/SortCommandTest.java]: indentation here should be just 4 spaces from previous line

File: test/java/seedu/addressbook/commands/SortCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    public static final String MESSAGE_SUCCESS = "All people sorted";
+    public static final String MESSAGE_ALREADY_SORTED = "Already sorted";
+
+    UniquePersonList tempPersonList = new UniquePersonList();
+    Person expectedPerson;
+
+    @Test
+    public void sortCommand_correctlySorted(){
+        try {
+            Person temp1 = new Person(
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/commands/SortCommandTest.java]: can possibly increase the SLAP in this method?

File: test/java/seedu/addressbook/commands/SortCommandTest.java (hunk 3 above)
Comment (eugenepeh) [reply test/java/seedu/addressbook/commands/SortCommandTest.java]: Sorry @philemontan, error in my review here. 

I've commented that the indentation should be 4 spaces from previous line but actually 
it should be the "+" operator on next line here.

Was looking through too quickly, sorry about that.

Comment (philemontan) [reply test/java/seedu/addressbook/commands/SortCommandTest.java]: Got it. Thanks!!

**Pull request closed without merging**